charts/wikijs/values.yaml

```diff
@@ -11,8 +11,6 @@ secret: {}
 env:
   # https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
   TZ: "America/Los_Angeles"
-  # PUID: "1000"
-  # PGID: "1000"
```

Comment:
These should be left as comments as they're environmental variables that people can use.

Reply:
These values are consumed by the image, as are several others; however, they shouldn't be modified or suggested to users of this chart as they're not relevant in the k8s context; changing them would also require you to change the security context.
I can't think of any use case which would require those values to be changed, hence I removed them.